docs/examples/tests/guides/test_backends_guide.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from orquestra.integrations.qiskit.backend import QiskitBackend
+from orquestra.quantum.backends import SymbolicSimulator
+from orquestra.quantum.circuits import Circuit, X
+
+from ...guides.backends_guide import MyBackend
+
+api_token = os.getenv("ZAPATA_IBMQ_API_TOKEN")
+
+
+def test_my_simulator_produces_output():
```

Comment:
I don't think we actually need this file at all?
It's enough that we make sure that the example code runs, right?

Reply:
The primary purpose of this file is to import variables from `backends_guide.py` so all the code gets run. 

The extra tests are just to deal with the `api_token` while keeping the code snippet the same. Mostly because I like the rick and morty joke ethan put in there. 😉